app-doc/doxygen/doxygen-1.8.12.ebuild

```diff
@@ -0,0 +1,153 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit cmake-utils eutils fdo-mime flag-o-matic python-any-r1
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/doxygen/doxygen.git"
+	SRC_URI=""
+	KEYWORDS="alpha amd64 arm hppa ia64 ppc ppc64 sparc x86"
```

Comment:
This code is not executed, but anyway the KEYWORDS should be empty. It would make sense to update with this PR also the 9999 version.